examples/index.json

```diff
@@ -975,7 +975,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Inferring isolated",
+                "name": "Infer isolated",
```

Comment:
For this and L950 also, I think what we had previously is better.

Reply:
Reverted both.